lib/codegen/Decision.cpp

```diff
@@ -1045,13 +1058,44 @@ void makeStepFunction(
   codegen(dt);
 }
 
+void makeMatchReasonFunctionWrapper(
+    KOREDefinition *definition, llvm::Module *module,
+    KOREAxiomDeclaration *axiom, std::string name) {
+  auto blockType = getValueType({SortCategory::Symbol, 0}, module);
+  llvm::FunctionType *funcType = llvm::FunctionType::get(
+      llvm::Type::getVoidTy(module->getContext()), {blockType}, false);
+  std::string wrapperName = "match_" + std::to_string(axiom->getOrdinal());
+  llvm::Function *matchFunc
+      = getOrInsertFunction(module, wrapperName, funcType);
+  std::string debugName = name;
+  if (axiom->getAttributes().count("label")) {
+    debugName = axiom->getStringAttribute("label") + "_fastcc_" + ".match";
```

Comment:
Do other usages respect this `_fastcc_` suffix? I don't see any other references to it in the code